apps/signup-form/src/components/pages/FormView.tsx

```diff
@@ -65,7 +65,8 @@ const Form: React.FC<FormProps> = ({isMinimal, loading, success, error, buttonCo
                     data-testid="input"
                     disabled={loading || success}
                     placeholder={t('Your email address')}
```

Comment:
Are we comfortable with this input not being associated with a label tag?

Reply:
Good point, that's tracked via [#1251](https://github.com/AssistivLabs/ghost-demo/issues/1251), we'll fix in a future PR.